.github/workflows/preview.yml

```diff
@@ -29,5 +31,10 @@ jobs:
       - name: Build
         run: npm run package
 
+      - name: Prepare pnpm for preview publishing
+        run: |
+          npm install -g corepack@latest
+          corepack prepare pnpm@latest --activate
```

Comment:
Why pnpm when we use npm?

Reply:
https://github.com/stackblitz-labs/pkg.pr.new/issues/370